galley/pkg/server/controller.go

```diff
@@ -29,37 +29,19 @@ const (
 	DefaultReadinessProbeFilePath = "/healthReadiness"
 )
 
-var (
-	livenessProbeController, readinessProbeController probe.Controller
-	galleyProbe                                       *probe.Probe
-)
-
 //StartProbeCheck start probe check for Galley
-func StartProbeCheck(livenessProbeOptions *probe.Options, readinessProbeOptions *probe.Options, stop <-chan struct{}) {
+func StartProbeCheck(livenessProbeController *probe.Controller, readinessProbeController *probe.Controller, stop <-chan struct{}) {
```

Comment:
`probe.Controller` is an interface and shouldn't be passed by pointer.

```golang
func StartProbeCheck(livenessProbeController, readinessProbeController probe.Controller, stop <-chan struct{}) {
```

Reply:
done